core/src/main/scala/no/nrk/bigquery/BQSqlFrag.scala

```diff
@@ -286,11 +287,24 @@ object BQSqlFrag {
             case Nil => None
           }
 
+        val fromRangePartitioned: Option[BQSqlFrag] =
+          partitions.collect { case x: BQPartitionId.RangePartitioned =>
+            x
+          }.sorted match {
+            case partitions@(first :: _) =>
+              val in = partitions.map(_.partition).mkFragment("[", ", ", "]")
```

Comment:
since we know the set here we dont have to go via array.
```suggestion
              val in = partitions.map(_.partition).mkFragment("(", ", ", ")")
```
then we can avoid the unnesting in the next fragment